Apps/Sandcastle/gallery/Clipping Planes.html

```diff
@@ -36,107 +36,170 @@
 <div id="cesiumContainer" class="fullSize"></div>
```

Comment:
Perhaps rename the examples to
* 3D Tiles Clipping Planes
* Terrain Clipping Planes